control-plane/pkg/reconciler/consumergroup/controller.go

```diff
@@ -167,10 +167,14 @@ func ResyncOnStatefulSetChange(ctx context.Context, handle func(interface{})) {
 
 func enqueueConsumerGroupFromConsumer(enqueue func(name types.NamespacedName)) func(obj interface{}) {
 	return func(obj interface{}) {
-		c := obj.(*kafkainternals.Consumer)
-		or := c.GetConsumerGroup()
-		if or != nil {
-			enqueue(types.NamespacedName{Namespace: c.GetNamespace(), Name: or.Name})
+		c, ok := obj.(*kafkainternals.ConsumerGroup)
+		if ok {
+			or := c.GetVReplicas()
+			if or != 0 {
+				enqueue(types.NamespacedName{Namespace: c.GetNamespace(), Name: "F"})
+			}
+		} else {
+			time.Sleep(50* time.Second)
 		}
```

Comment:
why sleep?

Reply:
removed - was for local debugs :man_facepalming: